pipelines/skylab/multiome/test_inputs/Scientific/10k_pbmc.json

```diff
@@ -1,6 +1,5 @@
 {
-"Multiome.annotations_gtf":"gs://broad-gotc-test-storage/Multiome/input/scientific/10k_PBMC_Multiome/introns_modified_gencode.vGencode_V43.primary_assembly.annotation.gtf",
-  "Multiome.barcodes_in_read_name":"false",
+"Multiome.annotations_gtf":"gs://broad-gotc-test-storage/Multiome/input/scientific/10k_PBMC_Multiome/modified_v43.annotation.gtf",
```

Comment:
when using the intronic GTF, it fails in optimus (calculateCellMetrics) with:
Expected 2 fields, found 0 fields
Expected 2 fields, found 0 fields

Reply:
note to self: change back to original --> gs://broad-gotc-test-storage/Multiome/input/scientific/10k_PBMC_Multiome/introns_modified_gencode.vGencode_V43.primary_assembly.annotation.gtf